tests/acceptance/window-test.js

```diff
@@ -12,4 +14,16 @@ module('Acceptance | window', function(hooks) {
 
     assert.equal(window.location.href, 'http://www.example.com/');
   });
+
+  sinonTest('it calls reset in the hooks', function(assert) {
```

Comment:
It seems this does not have to be in an acceptance test. You could move that to a dedicated unit test, like `tests/unit/setup-window-mock-test.js`.

Reply:
I've moved it into a unit test file